core/src/main/java/org/apache/iceberg/ManifestGroup.java

```diff
@@ -104,6 +127,29 @@ ManifestGroup caseSensitive(boolean newCaseSensitive) {
   }
 
   /**
+   * Returns a iterable of scan tasks. It is safe to add entries of this iterable
+   * to a collection as {@link DataFile} in each {@link FileScanTask} is defensively
+   * copied.
+   * @return a {@link CloseableIterable} of {@link FileScanTask}
+   */
+  public CloseableIterable<FileScanTask> scanTasks() {
```

Comment:
Should this be called `planFiles()`? That's what we use elsewhere when returning a collection of `FileScanTask`.